templates/apache2/mods-available/ssl.conf.j2

```diff
@@ -74,7 +74,7 @@
 	SSLHonorCipherOrder on
 
 	#   The protocols to enable.
-	#   Available values: all, SSLv3, TLSv1, TLSv1.1, TLSv1.2, TLSv1.3
```

Comment:
That looks surprising

Reply:
Yep, I thought so too.

Reply:
But at the moment you find the same change stretch and buster:

```
-       #   Available values: all, SSLv3, TLSv1, TLSv1.1, TLSv1.2
+       #   Available values: all, SSLv3, TLSv1, TLSv1.1, TLSv1.2, TLSv1.3